detections/endpoint/windows_sql_server_extended_procedure_dll_loading_hunt.yml

```diff
@@ -0,0 +1,49 @@
+name: Windows SQL Server Extended Procedure DLL Loading Hunt
+id: 182ba99f-2dde-4cdb-8e5c-e3b1e251cb10
+version: 1
+date: '2025-02-10'
+author: Michael Haag, Splunk
+status: production
+type: Hunting
+description: This analytic detects when SQL Server loads DLLs to execute extended stored procedures. This is particularly important for security monitoring as it indicates the first-time use or version changes of potentially dangerous procedures like xp_cmdshell, sp_OACreate, and others. While this is a legitimate operation, adversaries may abuse these procedures for execution, discovery, or privilege escalation.
+data_source:
+- Windows Event Log Application
+search: '`wineventlog_application` EventCode=8128
```

Comment:
Can we add a new data source object for this event Code and update the name in data_source